code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -174,7 +174,7 @@
 	loot = list(
 		/obj/item/organ/tongue = 5,
 		/obj/item/organ/tongue/lizard = 1,
-		/obj/item/organ/tail/cat = 1,
+		/obj/item/organ/tail/human/cat = 1,
```

Comment:
I see zero reason putting them into human/, as other organs do not have this kind of inheritance, this is pointless and only messes up the code more.